config/owasp/suppressions.xml

```diff
@@ -12,6 +12,10 @@
     <cve>CVE-2013-4152</cve>
     <cve>CVE-2014-0054</cve>
     <cve>CVE-2013-7315</cve>
+    <cve>CVE-2022-22968</cve>
+    <cve>CVE-2022-22968</cve>
+    <cve>CVE-2020-11022</cve>
+    <cve>CVE-2020-11023</cve>
```

Comment:
Should be able to revert these changes now as master has been updated.

Reply:
fixed in https://github.com/hmcts/hmc-hmi-inbound-adapter/pull/57/commits/d6c7102ac5ec829df9b005bb47e9e623bb26f9f9